.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Animal Farm NodeJS CI
```

Comment:
Really? haha

Maybe name it something relevant?

Reply:
Yeah - opened the PR that soon by mistake :)